### DISCUSSION TITLE
Home Assistant 2024.8

### DISCUSSION THREAD

**Discussion by stevetrease** [BottlecapDave/HomeAssistant-OctopusEnergy#965] [General]
Hello, 

Before I dive in and upgrade, has anyone confirmed the integration works with this week's release of Home Assistant 2024.8?

Cheers,

Steve

**Comment by 1st-Ade** [comment]
I have this working with Predbat-add-on on 2024.8

**Reply by stevetrease** [reply]
I upgraded yesterday and can confirm that I see no problems.
